src/App.jsx

```diff
@@ -14,33 +15,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const ALPHABET_SORT = 'alphabet';
+const STR_LENGTH_SORT = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [goodsList, setGoodsList] = useState(goodsFromServer);
+  const [rev, setRev] = useState(false);
+  const [classActive, setClassActive] = useState('');
+  const [reverseActive, setREverseActive] = useState(true);
```

Comment:
There is a typo in the state setter function name. It should be `setReverseActive` instead of `setREverseActive` to match the state variable `reverseActive`.